orchagent/portsorch.cpp

```diff
@@ -1869,6 +1870,22 @@ bool PortsOrch::addHostIntfs(Port &port, string alias, sai_object_id_t &host_int
     strncpy((char *)&attr.value.chardata, alias.c_str(), SAI_HOSTIF_NAME_SIZE);
     attrs.push_back(attr);
 
+    const char *platform = getenv("platform");
+    if (platform && strstr(platform, BRCM_PLATFORM_SUBSTRING))
```

Comment:
Platform specific checks are not allowed in PI code. There was a lengthy discussion last year and no more new checks are allowed other than the existing ones.